business-central-parent/business-central-distribution-wars/business-central/pom.xml

```diff
@@ -50,7 +50,7 @@
     <dependency>
       <groupId>org.kie</groupId>
       <artifactId>business-central-webapp</artifactId>
-      <type>jar</type>
+      <!--type>jar</type-->
```

Comment:
Is this intentionally commented?

Reply:
yes, i looked at cargo maven documentation page and there is no Thorntail container yet and as The old Wildfly swarm container seems to work I still keep the profile as WF swarm as it uses config with such property string names. Not ideal, but better to point out as I tried to change names and it broke. So keeping it intentionally named like that until we have Thorntail container.

Reply:
yes, i tried to not use default type ;)